java-persistence/src/main/java/com/whitecloak/training/inventory/service/impl/CategoryServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package com.whitecloak.training.inventory.service.impl;
+
+import com.whitecloak.training.inventory.controller.request.CategoryForm;
+import com.whitecloak.training.inventory.controller.response.CategoryResource;
+import com.whitecloak.training.inventory.gateway.CategoryGateway;
+import com.whitecloak.training.inventory.model.Category;
+import com.whitecloak.training.inventory.service.CategoryService;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class CategoryServiceImpl implements CategoryService {
```

Comment:
This class has no `@Component` annotation (or its derivatives).

This component will not work.